test/src/test/java/jenkins/model/JenkinsBuildsAndWorkspacesDirectoriesTest.java

```diff
@@ -258,6 +292,15 @@ private boolean logWasFound(String searched) {
                 .anyMatch(record -> record.getMessage().contains(searched));
     }
 
+	private boolean logLevelWasFound(String searched, Level level) {
+		return loggerRule
+				.getRecords()
+				.stream()
+				.filter(record -> record.getMessage().contains(searched)
+						&& record.getLevel().equals(level))
```

Comment:
For lambda clarity/declarativeness, I'd recommend putting two filters instead of using `&&`:

```java
.filter(record -> record.getMessage().contains(searched))
.filter(record -> record.getLevel().equals(level))
```